artemis-server/src/main/java/org/apache/activemq/artemis/spi/core/security/jaas/kubernetes/client/KubernetesClientImpl.java

```diff
@@ -160,9 +163,13 @@ private SSLContext buildSSLContext() throws Exception {
       try (InputStream fis = new FileInputStream(certFile)) {
          KeyStore trustStore = KeyStore.getInstance(KeyStore.getDefaultType());
```

Comment:
I think it would make sense to lean on the PEM keyStoreType support, and make use of SSLSupport utility calss to load a trust store. In that way, this ssl client will behave like the others in the broker.
When this client was introduced there was no support for PEM  in the broker, but now there is.

Reply:
I'm trying that out, but without much success actually. It doesn't seem to be able to load the `ca.crt` file properly.